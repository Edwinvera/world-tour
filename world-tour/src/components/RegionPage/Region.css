.AsiaRegion{
  background-image: url("../../images/RegionRegions/AsiaRegion.jpg");
  background-repeat: no-repeat;
  background-size: cover;
}
.AfricaRegion{
  background-image: url("../../images/RegionRegions/AfricaRegion.jpg");
  background-repeat: no-repeat;
  background-size: cover;
}
.EuropeRegion{
  background-image: url("../../images/RegionRegions/EuropeRegion.jpg");
  background-repeat: no-repeat;
  background-size: cover;
}
.AmericasRegion{
  background-image: url("../../images/RegionRegions/AmericasRegion.jpg");
  background-repeat: no-repeat;
  background-size: cover;
}
.PolarRegion{
  background-image: url("../../images/RegionRegions/PolarRegion.jpg");
  background-repeat: no-repeat;
  background-size: cover;
}
.OceaniaRegion{
  background-image: url("../../images/RegionRegions/OceaniaRegion.jpg");
  background-repeat: no-repeat;
  background-size: cover;
}