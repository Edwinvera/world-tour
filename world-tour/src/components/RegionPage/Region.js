import React from 'react'
import { Link } from 'react-router-dom'

function Region() {
  return (
    <div>
      <main>
        <h1>name of Region clicked</h1>
        <hr/>
        <table>
          <tbody>
            <tr>
              <td>Population in *name here*</td>
              <td>########</td>
            </tr>
            <tr>
              <td>Area of Land (mi<sup>2</sup>)</td>
              <td>#####</td>
            </tr>
            <tr>
              <td>Total Countries</td>
              <td>####</td>
            </tr>
          </tbody>
        </table>
        <hr />
        {/* Need to setup a map loop for regions here*/}
        <div>
          <Link to='/country/:alpha3code'>
            Botwana
          </Link>
        </div>
      </main>
    </div>
  )
}

export default Region