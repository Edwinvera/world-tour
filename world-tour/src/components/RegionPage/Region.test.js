import renderer from 'react-test-renderer'
import 'jest-styled-components'
import React from 'react'
import { Table, CountryList, Country }from './Region'

test('Region_Table component renders', () => {
  const TableComponent = renderer.create(<Table/>).toJSON()
  expect(TableComponent).toMatchSnapshot()
})

test('Region_CountryList component renders', () => {
  const CountryListComponent = renderer.create(<CountryList/>).toJSON()
  expect(CountryListComponent).toMatchSnapshot()
})

test('Region_Country component renders', () => {
  const CountryComponent = renderer.create(<Country/>).toJSON()
  expect(CountryComponent).toMatchSnapshot()
})
