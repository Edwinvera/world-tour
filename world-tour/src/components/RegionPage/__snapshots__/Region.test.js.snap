// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Region_Country component renders 1`] = `
.c0 {
  font-family: 'Nunito Sans',sans-serif;
  font-size: 18px;
  -webkit-letter-spacing: 5px;
  -moz-letter-spacing: 5px;
  -ms-letter-spacing: 5px;
  letter-spacing: 5px;
  color: rgb(60,60,155);
}

@media (max-width:1050px) {
  .c0 {
    font-size: 25px;
    line-height: 3;
  }
}

<h2
  className="c0"
/>
`;

exports[`Region_CountryList component renders 1`] = `
.c0 {
  overflow: auto;
  height: 615px;
  background-color: rgba(10,10,10,.1);
  padding-left: 15px;
}

@media (max-width:1050px) {
  .c0 {
    height: 550px;
  }
}

<section
  className="c0"
/>
`;

exports[`Region_Table component renders 1`] = `
.c0 {
  width: 400px;
}

@media (max-width:1050px) {
  .c0 {
    width: 800px;
    text-align: left;
  }
}

@media (max-width:930px) {
  .c0 {
    width: 500px;
    text-align: left;
  }
}

@media (max-width:697px) {
  .c0 {
    width: 300px;
    text-align: left;
  }
}

<table
  className="c0"
/>
`;
